src/App.jsx

```diff
@@ -4,32 +4,26 @@ import viteLogo from '/vite.svg'
 import './App.css'
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [num, setNum] = useState(0);
 
+  function randomNumberInRange(min, max) {
+    // 👇️ Get the number between min (inclusive) and max (inclusive)
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+
+  const [num2, setNum2] = useState(0);
+  const handleClick = () => {
+    setNum(randomNumberInRange(1, 5));
+    setNum2(randomNumberInRange(1, 5));
+  };
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+    <div>
+      <h2>dice 1: {num}</h2>
+      <h3>dice 2: {num2}</h3>
```

Comment:
```suggestion
      <div>dice 1: {num}</div>
      <div>dice 2: {num2}</div>
```
h1, h2 und h3 sind eigentlich nur für headlines gedacht, in diesem Fall ist es besser etwas wie div oder span zu benutzen und dann das styling später mit css anzupassen